modules/ROOT/pages/runtime-fabric-limitations.adoc

```diff
@@ -47,8 +47,12 @@ The following table lists known limitations for host operating systems in Runtim
 [%header%autowidth.spread]
 |===
 | Operating System | Limitation
-| RHEL 8 | If the host operating system is RHEL 8, do not set the optional environment variable `disable_selinux` to `false` in the `fabric.tf` script, or installation will fail.
-| RHEL 7 | If the host operating system is RHEL 7, and you have enabled SELinux, third-party software running on the host may change policy settings from SELinux.
+| RHEL 8 a| 
+* Do not set the optional environment variable `disable_selinux` to `false` in the `fabric.tf` script, or installation will fail.
+* Do not set SELinux to enforcing mode. 
+| RHEL 7 a| 
+* If you have enabled SELinux, third-party software running on the host may change policy settings from SELinux.
+* You can set SELinux to enforcing mode if you don't change default policies.
```

Comment:
Maybe this? I know we usually try to avoid modal verbs like "can."
```suggestion
* Runtime Fabric is compatible with unchanged default policies and SELinux set to enforcing mode.
```